controllers/virtualmachine/volumebatch/volumebatch_controller.go

```diff
@@ -1269,3 +1252,20 @@ func categorizeVolumeSpecs(
 
 	return volumeSpecsForBatch, volumeSpecsForLegacy
 }
+
+// updateVolumeStatusWithExistingVMStatus updates the target vmVolStatus with
+// info from existing VM managed volume status.
+func updateVolumeStatusWithExistingVMStatus(
+	vmVolStatus *vmopv1.VirtualMachineVolumeStatus,
+	existingVMManagedVolStatusMap map[string]vmopv1.VirtualMachineVolumeStatus) {
+
+	existingVolStatus := existingVMManagedVolStatusMap[vmVolStatus.Name]
```

Comment:
Just glancing at this MR - will this item always exist and will be `Type` be set?

Reply:
Good catch! Not it won't.
But it won't error since `vmopv1.VirtualMachineVolumeStatus` is not a pointer.
I can add a check here to skip below assignment

Reply:
Is this worth covering in a test?

Reply:
It's already covered 
<img width="637" height="376" alt="image" src="https://github.com/user-attachments/assets/32a6f322-95c8-47cb-b34d-9669112e0f7b" />

Reply:
Ok i removed it since you mentioned it's not necessary